applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -789,8 +894,13 @@ static void unicast_client_location_cb(struct bt_conn *conn, enum bt_audio_dir d
 	struct le_audio_unicast_server *unicast_server =
 		&unicast_servers[idx.lvl1][idx.lvl2][idx.lvl3];
 
-	if ((loc & BT_AUDIO_LOCATION_FRONT_LEFT) || (loc & BT_AUDIO_LOCATION_SIDE_LEFT) ||
-	    (loc == BT_AUDIO_LOCATION_MONO_AUDIO)) {
+	if (loc & BT_AUDIO_LOCATION_FRONT_LEFT && loc & BT_AUDIO_LOCATION_FRONT_RIGHT) {
+		LOG_INF("Both front left and right channel locations are set, stereo device found");
+		unicast_server->location =
+			BT_AUDIO_LOCATION_FRONT_LEFT | BT_AUDIO_LOCATION_FRONT_RIGHT;
+		unicast_server->ch_name = "STEREO";
+	} else if ((loc & BT_AUDIO_LOCATION_FRONT_LEFT) || (loc & BT_AUDIO_LOCATION_SIDE_LEFT) ||
+		   (loc == BT_AUDIO_LOCATION_MONO_AUDIO)) {
```

Comment:
I see that this was also the case before, but why are we considering `BT_AUDIO_LOCATION_FRONT_LEFT` and `BT_AUDIO_LOCATION_SIDE_LEFT` as LEFT? 

What about the other `_LEFT` values?